fedimg/services/ec2/ec2imguploader.py

```diff
@@ -24,6 +24,8 @@
 
 import re
 
+from time import sleep
+
```

Comment:
i don't understand why this is removed. things in this file use it

Reply:
This was not removed, this was added.

Reply:
oops. i guess my diff view confused me on that. Can you put this in a separate commit?